### PR TITLE
executor: refine point get failpoint injection logic

### PR DESCRIPTION
**Pull request by lonng**
Signed-off-by: Lonng <chris@lonng.org>

### What problem does this PR solve? 

Refine point get fault injection logic

### What is changed and how it works?

1. Change the failpoint expression from `return(true)` to `return` and remove `debugger`
2. Refine the failpoint closure.

### Check List 

Tests 

 - Unit test
 - Integration test

Code changes

N/A

Side effects

Related changes

### CODE REVIEW

File: executor/executor_test.go [hunk 1]
```diff
@@ -1888,19 +1884,33 @@ func (s *testSuite) TestPointGetRepeatableRead(c *C) {
...
-	label := debugger.Bind("point-get-g2")
-	debugger.Continue("point-get-g1")
-	debugger.Breakpoint(label)
+	defer func() {
+		c.Assert(failpoint.Disable(step1), IsNil)
+		c.Assert(failpoint.Disable(step2), IsNil)
+	}()
+
+	<-updateWaitCh // Wait `POINT GET` first time `get`
 	tk2.MustExec("update point_get set b = 2, c = 2 where a = 1")
```
Comment (tiancaiamao) [inline executor/executor_test.go]: `failpoint.Disable(step2)` should be here?

Comment (tiancaiamao) [reply executor/executor_test.go]: After this line?

Comment (lonng) [reply executor/executor_test.go]: Yes, `step2` means point get wait for `UPDATE` finish and defer will execute after L1913.

Comment (lonng) [reply executor/executor_test.go]: I have reorganized the part, PTAL again.

**Comment by tiancaiamao** [comment]
LGTM

**Comment by lonng** [comment]
@winkyao @jackysp PTAL

**Pull request merged**